basement/Basement/Alg/Native/UTF8.hs

```diff
@@ -125,6 +126,16 @@ prevSkip ba offset = loop (offset `offsetMinusE` sz1)
         | isContinuation (primIndex8 ba o) = loop (o `offsetMinusE` sz1)
         | otherwise                       = o
 
+sizeChar :: Char -> CountOf Word8
+sizeChar !c
```

Comment:
might be a good idea to re-use/re-purpose basement/Basement/UTF8/Helper.hs `charToByte`